src/components/app/details/cIDetails/CIDetails.tsx

```diff
@@ -669,16 +667,13 @@ export const NoVulnerabilityViewWithTool = ({ scanToolId }: { scanToolId: number
 const SecurityTab = ({ ciPipelineId, artifactId, status, appIdFromParent }: SecurityTabType) => {
     const { appId } = useParams<{ appId: string }>()
     const { push } = useHistory()
-    const isSecurityScanV2Enabled = window._env_.ENABLE_RESOURCE_SCAN_V2 && !!isFELibAvailable
 
     const computedAppId = appId ?? appIdFromParent
 
-    const { scanDetailsLoading, scanResultResponse, executionDetailsResponse, scanDetailsError, reloadScanDetails, severityCount, totalCount } =
-        useGetCISecurityDetails({
-            appId: +computedAppId,
-            artifactId,
-            isSecurityScanV2Enabled,
-        })
+    const [scanResultLoading, scanResultResponse, scanResultError, reloadScanResult] = useAsync(
```

Comment:
We should use `useGetAppSecurityDetails` instead of directly calling the method. or we should delete `useGetAppSecurityDetails` and use like this everywhere